app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -724,6 +729,37 @@ public UploadResult uploadFile(String filename,
         });
     }
 
+    /**
+     * This takes userName as input, which is then used to fetch the no of images deleted
+     * using OkHttp and JavaRx. This function return JSONObject
+     * @param userName
+     * @return
+     */
+    @NonNull
+    @Override
+    public Single<JSONObject> getRevertCount(String userName){
```

Comment:
- Now that, we have so many similar calls to `urbanecmbot/commonsmisc/feedback.py` it might be better to refactor it to a common function. 

- The function name doesn't convey that the return type would be a `Single<JsonObject>`. If you wish to return `Single<JsonObject>` then a more appropriate function name would be `getRevertRespObjectSingle`. Or otherwise, if you refactor the function to just return the count instead of the whole `JsonObject` then the signature could be:

```
public Single<Integer> getRevertCount(String userName)
```